src/server_manager/web_app/ui_components/outline-intro-step.js

```diff
@@ -290,6 +313,10 @@ Polymer({
     return Boolean(accountName);
   },
 
+  _showNewGcpFlow(gcpAccountName) {
+    return outline.gcpAuthEnabled || this._computeIsAccountConnected(gcpAccountName);
```

Comment:
Let's remove the `outline.gcpAuthEnabled` check. If an account is connected, we should always show the new flow.

This check should be in the manual screen instead, to control when to show the experimental flow escape hatch.

Reply:
> If an account is connected, we should always show the new flow.

We do.  (This is an `||`.)  The purpose of checking `gcpAuthEnabled` is to show the magic-flow promo card in the logged-out state.

> This check should be in the manual screen instead, to control when to show the experimental flow escape hatch.

The manual screen for GCP is unreachable when `gcpAuthEnabled` is true, so that seems moot.

Reply:
You're right, I misread it.